docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>i think it's more natural to differentiate <code>reg_ot_cost</code> using Danskin (differentiating <code>primal_cost</code> triggers implicit differentiation)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>